src/main/java/htsjdk/samtools/cram/encoding/DataSeries.java

```diff
@@ -1,49 +0,0 @@
-/**
```

Comment:
The new `DataSeries` enum now contains the association between the old `EncodingKey` and `DataSeriesType`